.github/workflows/ci.yml

```diff
@@ -98,28 +99,34 @@ jobs:
         run: mkdir -p ember-core/js/target ember-server/js/target examples/ember-server-h2/target website/target examples/ember/target examples/ember-client-h2/target blaze-client/target server/jvm/target examples/target scalafix-internal/input/target blaze-server/target scalafix-internal/tests/target scalatags/target target examples/docker/target twirl/target .js/target docs/target laws/.jvm/target tests/jvm/target client/js/target tomcat-server/target prometheus-metrics/target scalafix-internal/output/target servlet/target examples/tomcat/target examples/jetty/target boopickle/.js/target testing/jvm/target core/js/target testing/js/target ember-server/jvm/target circe/.js/target okhttp-client/target laws/.js/target examples/blaze/target jawn/.js/target tests/js/target .jvm/target scalafix-internal/rules/target .native/target client/jvm/target blaze-core/target circe/.jvm/target dsl/.js/target examples/war/target jetty-client/target ember-client/jvm/target boopickle/.jvm/target play-json/target async-http-client/target core/jvm/target scala-xml/target dsl/.jvm/target dropwizard-metrics/target jetty-server/target node-serverless/target ember-core/jvm/target jawn/.jvm/target server/js/target ember-client/js/target bench/target project/target
 
       - name: Compress target directories
+        if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/main')
         run: tar cf targets.tar ember-core/js/target ember-server/js/target examples/ember-server-h2/target website/target examples/ember/target examples/ember-client-h2/target blaze-client/target server/jvm/target examples/target scalafix-internal/input/target blaze-server/target scalafix-internal/tests/target scalatags/target target examples/docker/target twirl/target .js/target docs/target laws/.jvm/target tests/jvm/target client/js/target tomcat-server/target prometheus-metrics/target scalafix-internal/output/target servlet/target examples/tomcat/target examples/jetty/target boopickle/.js/target testing/jvm/target core/js/target testing/js/target ember-server/jvm/target circe/.js/target okhttp-client/target laws/.js/target examples/blaze/target jawn/.js/target tests/js/target .jvm/target scalafix-internal/rules/target .native/target client/jvm/target blaze-core/target circe/.jvm/target dsl/.js/target examples/war/target jetty-client/target ember-client/jvm/target boopickle/.jvm/target play-json/target async-http-client/target core/jvm/target scala-xml/target dsl/.jvm/target dropwizard-metrics/target jetty-server/target node-serverless/target ember-core/jvm/target jawn/.jvm/target server/js/target ember-client/js/target bench/target project/target
 
       - name: Upload target directories
+        if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/main')
```

Comment:
A minor but truly grouchy nitpick: reference to the `main` branch is a little bit confusing within `series/0.22`/`series/0.23`. Would CI publish snapshots for the `series/0.22`/`series/0.23` at all?

Reply:
I'm grouchy too. We can't publish snapshot until the website is fixed.
- https://github.com/http4s/http4s/pull/5920

Reply:
Yeah, and those patches that we're talking about are going to break things again.  Getting that fixed needs to be a priority, but I doubt I'll have time to investigate before Friday.